### DISCUSSION TITLE
Zed Honest Obersvation

### DISCUSSION THREAD

**Discussion by IsaacPere** [zed-industries/zed#15861] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Good Evening from Kenya
I am of the strongest belief that zed-industries has fallen into a rabbit hole of what can work on Mac but also work on Linux, first of all  that thinking is damaging enough for it pressures the developers and the entire community to develop features and tools while at the same time forgoing the fundamental principles of kiss that is keeping the application simple.
It makes me to wonder that having the advantage of using the language rust  which is known for it compatibility, usability and memory safety that enables for errors to be caught at the soonest, the IDE keeps on crashing, blinking as if it has been posses by a virus.  Please can the community members take time to develop features that would run well on different flavors of Linux for example I haven't seen a Debian based distribution channel that would developers who are using Ubuntu, Kali and many more distributions to work with the IDE which will result the community  to correct errors and bugs that would emerge from using IDE.
It may seems that I am harsh but what has betrayed the IDE from me view has been that the IDE was rushed very fast  but if it could have been developed without the rush, features bugs like auto open of projects would not be the problem that most of the developers would have been complaining and comparing it to Electron, a framework that is known to be buggy and slow to adapt to change and innovation.   

### If applicable, add mockups / screenshots to help present your vision of the feature

The piece of advice that I would like to point it to the members can we redevelop the IDE slowly and also test the features that we are presenting with the IDE, for example making it be compatible with different versions of Linux. Slow and steady until we achieve the advantage of the IDE

**Comment by apricotbucket28** [comment]
> the IDE keeps on crashing, blinking as if it has been posses by a virus

Could you report those crashes (along with backtraces)? Saying that you experience [*some vague issue*] won't help you get it solved.

> Please can the community members take time to develop features that would run well on different flavors of Linux

What doesn't run well on different flavours of Linux? The Zed team mainly tests on Ubuntu but I use it on Fedora without any issues.
The only issue I can think of is downloading glibc-compiled LSP binaries on distros like Alpine (see https://github.com/zed-industries/zed/issues/12589)

> I haven't seen a Debian based distribution channel that would developers who are using Ubuntu, Kali

Even though there's no .deb package you can still install via the `curl | sh` script or [Flathub](https://flathub.org/apps/dev.zed.Zed). If you'd like to support a deb package there's this issue: https://github.com/zed-industries/zed/issues/13952.

> bugs like auto open of projects

You mean https://github.com/zed-industries/zed/issues/7759?

> can we redevelop the IDE slowly and also test the features

1) That's unnecessary.
2) There's already a lot of automated tests and also manual testing (via the preview and nightly versions)
